README.md

```diff
@@ -1,5 +1,14 @@
+<a href="https://github.com/hypothesis/pyramid-sanity/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/workflow/status/hypothesis/pyramid-sanity/CI/main"></a>
+<a href="https://pypi.org/project/pyramid-sanity"><img src="https://img.shields.io/pypi/v/pyramid-sanity"></a>
+<a><img src="https://img.shields.io/badge/python-3.9 | 3.8-success"></a>
```

Comment:
Why is this a link? Does it need to be for styling or something? Clicking on it in GH shows the badge again in a new window.

Reply:
IIRC that was necessary to get all the badges to lay out horizontally in one row. Without the `<a>` it for some reason breaks onto the next row

Reply:
I wonder if we can add an attr to the `<a>` to prevent it from being clickable...

Reply:
Hmm. In the rendered HTML GitHub seems to have added an `href` to the `<a>` for us